src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -186,25 +232,98 @@ struct jit_has_subnormals : public jit_has_subnormals_base {
 
         copy_floats(r8, reg_src, reg_sz);
         check_subnormals(r8, exponent_mask, mantissa_mask, zero);
-        jc(no_subnormals);
+        jc(no_target_values);
         add(rsp, vlen * sizeof(float));
 
-        L(has_subnormals);
+        L(has_target_values);
 
         mov(rax, 1);
         mov(byte[reg_dst], al);
         jmp(exit);
 
-        L(no_subnormals);
+        L(no_target_values);
         add(rsp, vlen * sizeof(float));
 
         L(exit);
 
         postamble();
     }
 };
+template <cpu_isa_t isa>
+struct jit_has_bf16_overflows : public jit_has_special_value_base {
```

Comment:
It makes sense to merge `jit_has_bf16_overflows` and `jit_has_subnormals` functions into single configurable kernel. It will allow to check subnormals and overflows via single pass over the memory. It will also remove a significant amount of copy-paste inside jit kernel itslef and inside the code which calls it.

Reply:
Hi, @dmitry-gorokhov : about this concern, I have also discussed with @usstq offline. since these two `jit_has_bf16_overflows` and `jit_has_subnormals` check process mainly memory bound. how about we put this two process in the same parallel process(batch_size <= 2048)(the implement is updated in pr) which will make the processed source data always in cache? then we can keep this two `jit_has_bf16_overflows` and `jit_has_subnormals` check seperate and simple logic, and also reduce memory access bandwidth.

Reply:
I still do not agree that merging both checks inside single kernel makes jitter logic more complex. JIT approach allows to implement it in clear enough way.
Besides that using L1 as temporal storage between the checks still not the same as registers from perf perspective.
Anyway I don't want to post-pone this PR anymore. Lets do refactor in follow-up PRs (probably as part of BF16 saturation fusion with Reorder primitive).

Reply:
sure, will do refactor(merge `jit_has_bf16_overflows` and `jit_has_subnormals` kernels into one) in follow-up PRs